nixos/modules/system/activation/activation-script.nix

```diff
@@ -158,8 +158,11 @@ in
         ${pkgs.utillinux}/bin/mount -o "remount,size=${config.boot.devSize}" none /dev
         ${pkgs.utillinux}/bin/mount -o "remount,size=${config.boot.devShmSize}" none /dev/shm
         ${pkgs.utillinux}/bin/mount -o "remount,size=${config.boot.runSize}" none /run
+      '' + optionalString config.boot.tmpOnTmpfs ''
+        if [ mountpoint -q /tmp ]; then
```

Comment:
So what happens if a user says `boot.tmpOnTmpfs = true` but `/tmp` is not a valid mount point? The current solution is to fail silently, but that doesn't seem ideal?